api/all/src/main/java/io/opentelemetry/api/metrics/NoopMeter.java

```diff
@@ -56,7 +33,7 @@
  * </ul>
  */
 @ThreadSafe
-public class NoopMeter implements Meter {
+class NoopMeter implements Meter {
```

Comment:
What do you think about renaming these to `DefaultMeter` and `DefaultTracerProvider` to mirror tracing? Not super important because they're package private, but lowers the cognitive load if someone goes and comparing tracing to metrics.